Sources/carton/Commands/Dev.swift

```diff
@@ -69,9 +74,8 @@ struct Dev: ParsableCommand {
 
     let paths = try toolchain.inferSourcesPaths()
 
-    if !verbose {
-      terminal.clearWindow()
-      terminal.homeAndClear()
+    if !verbose {    
```

Comment:
<!--
  1 failure:  Lines should not ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/carton/Commands/Dev.swift;
  Line: 77;
-->
- :no_entry_sign: Lines should not have trailing whitespace. (`trailing_whitespace`)